packs/orion/aliases/node_status.yaml

```diff
@@ -10,11 +10,13 @@ formats:
 ack:
   enabled: false
   append_url: false
-  format: "Let me check {{execution.parameters.node}} on {{execution.parameters.node}} Orion Platform for you..."
+  format: "Let me check {{execution.parameters.node}} on {{execution.parameters.platform}} Orion Platform for you..."
 result:
+  extra:
+    color: "{{execution.result.result.color|default('#439FE0')}}"
   format: |
     {% if execution.status == 'succeeded' %}
-    {{ execution.parameters.node }} is {{ execution.result.result.status }}{~}
```

Comment:
I removed the separator so the whole message is in the attachment and you can see color. I'll drop a example on slack..